src/client/services/server-api/index.js

```diff
@@ -28,6 +28,20 @@ const ServerAPI = {
     return fetch(`/pc-client/querySearch?${qs.stringify(query)}`, defaultFetchOpts).then(res => res.json());
   },
 
+  uniprotId(query){
+  return fetch(`/pc-client/querySearch?${qs.stringify(query)}`, defaultFetchOpts).then(res => res.json()).then(res=> {
+    return _.compact(res.map(hit=>{
+        if( hit.uri.startsWith('http://identifiers.org/uniprot/') && _.endsWith(hit.organism[0],'9606')){
+         return _.last(hit.uri.split('/'));
+        }
+      }));
+    });
+  },
+
+  landingBox(query){
```

Comment:
Does this function get protein information based on some query args?

Reply:
It gets the protein information from UniProt based on the id that is passed in.

Reply:
Okay in that case it might make sense to rename this to

```js
getProteinInformation(uniprotId)
```
1.  ```landingBox``` implies that this function will only be used for the landing box but really anything could potentially want this information.  

2. ```query``` is a pretty general argument name.  In js it is more important to be a bit more specific with names.